x-pack/plugins/maps/public/index.js

```diff
@@ -70,7 +71,9 @@ routes
     template: mapTemplate,
     controller: 'GisMapController',
     resolve: {
-      map: function (gisMapSavedObjectLoader, redirectWhenMissing) {
+      map: function (gisMapSavedObjectLoader, redirectWhenMissing, Private) {
+        const docTitle = Private(DocTitleProvider);
+        docTitle.change("Maps");
```

Comment:
Why does the route `/map` set the tab name to `Maps`? Shouldn't this be the new map title `New Map`?

Reply:
I considered that, but it just seemed like tab clutter to me until they actually save the map. I'll check into the renaming issue!

Reply:
re: `/` route. Yeah, I don't know. I debated that too. Again it seemed like tab clutter to me so I opted not to. As a user that wouldn't help me much. Distinguishing between different saved maps is helpful though. Feel free to push back if you feel strongly though

Reply:
Dashboard does not set the tab name until the dashboard is saved. So maybe just keeping it `Kibana` is inline with other apps. Same with the `/` route. Maybe the only route that should effect the title is `/map/:id`

Reply:
So i guess the behavior right now isn't quite the same as discussed here:

- `Maps - Kibana` when creating a new map
- `[Map Name] Maps - Kibana` when viewing  a saved map. This one gets ugly when viewing saved objects for the sample data. e.g. `[[Flights] Origin and Destination Flight Time] Maps - Kibana`. Those brackets are unnecessary, as it the `Maps` name

Both are inconsistent with Dashboard and Visualize. 

I'd agree to keep it inline with the other apps.